rules/promotions/privilege_escalation_endgame_permission_theft_detected.toml

```diff
@@ -19,10 +19,17 @@ name = "Permission Theft - Detected - Elastic Endgame"
 risk_score = 73
 rule_id = "c3167e1b-f73c-41be-b60b-87f4df707fe3"
 severity = "high"
-tags = ["Elastic", "Elastic Endgame"]
+tags = ["Elastic", "Elastic Endgame", "Threat Detection", "Privilege Escalation"]
 type = "query"
 
 query = '''
 event.kind:alert and event.module:endgame and endgame.metadata.type:detection and (event.action:token_protection_event or endgame.event_subtype_full:token_protection_event)
 '''
 
+[[rule.threat]]
```

Comment:
maybe https://attack.mitre.org/techniques/T1134/ is more accurate ?

Reply:
added as the technique.